src/platform-includes/distributed-tracing/custom-instrumentation/dotnet.mdx

```diff
@@ -0,0 +1,64 @@
+On this page you will learn how to manually propagate trace information into and out of your .NET application. Please note, that you do not need to do this manually, if you [use one of our supported frameworks](/platforms/dotnet/usage/distributed-tracing/#how-to-use-distributed-tracing), or you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
```

Comment:
>  you do not need to do this manually, if you [use one of our supported frameworks](/platforms/dotnet/usage/distributed-tracing/#how-to-use-distributed-tracing)

This link doesn't make sense for this scenario, since the "How to Use Distributed Tracing?" section doesn't say anything about frameworks that automatically support distributed tracing. (For example, like Java's does https://docs.sentry.io/platforms/java/usage/distributed-tracing/#how-to-use-distributed-tracing)

Maybe rework this this paragraph to something like:

```suggestion
On this page you will learn how to manually propagate trace information into and out of your .NET application. Please note, that you do not need to do this manually if you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
```

Reply:
Or if there are frameworks or another way that automatically supports this, modify that section: https://sentry-docs-git-feat-distributed-trace-dotnet.sentry.dev/platforms/dotnet/usage/distributed-tracing/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-docs%2Fpull%2F8024#how-to-use-distributed-tracing

Reply:
Thanks for the catch. I updated that part as well.